### PR TITLE
feat(deploy): add dopplerci configuration

### PR DESCRIPTION
**Pull request by cbernat**
Add dopplerci configuration as [in MSEditor](https://github.com/MakingSense/MSEditor/pull/1446)

### CODE REVIEW

**Comment by cbernat** [comment]
@andresmoschini

File: .dopplerci [hunk 1]
```diff
@@ -0,0 +1 @@
+Jenkinsfile
```
Comment (andresmoschini) [inline .dopplerci:1]: The name is `.doppler-ci`, and ensure that it is a symlink to a valid Jenkinsfile:

> In Windows (with git bash as administrator): `export MSYS=winsymlinks:nativestrict; ln -s Jenkinsfile .doppler-ci`

See https://github.com/FromDoppler/hello-microservice#prepare-the-files

**Comment by cbernat** [comment]
Test this please.

**Pull request merged**